app/src/main/java/com/github/se/cyrcle/ui/map/MapScreen.kt

```diff
@@ -78,7 +83,8 @@ fun MapScreen(
     parkingViewModel: ParkingViewModel,
     userViewModel: UserViewModel,
     mapViewModel: MapViewModel,
-    zoomState: MutableState<Double> = remember { mutableDoubleStateOf(defaultZoom) }
+    zoomState: MutableState<Double> = remember { mutableDoubleStateOf(defaultZoom) },
+    permissionGranted: Boolean
```

Comment:
For Backward compatibility with all our old test, you should have put a default value of false here for permissionGranted.

This would have saved you from going modifying all of our actual test that use a map

Reply:
Yess i will make sure to change that when fixing the bug thanks